### PR TITLE
Fix press in classes to work with combo keys

### PR DESCRIPTION
**Pull request by ankur22**
`ElementHandle`, `Page`, `Locator` and `Frame` all implement the `Press` method. In a recent fix (#326) combo keys with the `Shift` modifier key was fixed for the `Keyboard` `Press` method. The `Press` methods for the other classes should have worked with combo key presses, but didn't, which resulted in an error: `ERRO[0002] pressing "Shift+KeyH+i" on "#text1": key down: "Shift+KeyH+i" is not a valid key for layout "us"`. This fix will ensure that combo key presses work for all of them. This fix matches the behaviour in playwright.

Playwright test:

```js
const { test, chromium } = require('@playwright/test');

test.describe('editing', () => {
  test('frame press', async () => {
    const browser = await chromium.launch({ headless: false, slowMo: 500 });
    const ctx = await browser.newContext();
    const page = await ctx.newPage();

    await page.goto("https://test.k6.io/browser.php");

    const e = await page.$('#text1');
    e.press("Shift+KeyH+i");
    await clearField(page);

    page.press('#text1', "Shift+KeyH+i");
    await clearField(page);

    const l = page.locator('#text1');
    l.press("Shift+KeyH+i");
    await clearField(page);

    const f = page.frames()[0];
    f.press('#text1', "Shift+KeyH+i");
    await clearField(page);

    await new Promise(resolve => setTimeout(resolve, 2000));

    await browser.close();
  });
});

async function clearField(page) {
  await new Promise(resolve => setTimeout(resolve, 500));
  page.press('#text1', 'Backspace');
  page.press('#text1', 'Backspace');
  await new Promise(resolve => setTimeout(resolve, 500));
}
```

Equivalent xk6-browser test:

```js
import { sleep } from 'k6'
import { chromium } from 'k6/x/browser';

export default function () {
  const browser = chromium.launch({
    headless: false,
  });
  const context = browser.newContext();
  const page = context.newPage();

  page.goto('https://test.k6.io/browser.php');

  const e = page.$('#text1');
  e.press("Shift+KeyH+i");
  clearField(page);

  page.press('#text1', "Shift+KeyH+i");
  clearField(page);

  const l = page.locator('#text1');
  l.press('Shift+KeyH+i');
  clearField(page);

  const f = page.frames()[0];
  f.press('#text1', 'Shift+KeyH+i');
  clearField(page);

  sleep(2);
  browser.close();
}

function clearField(page) {
  sleep(1);
  page.press('#text1', 'Backspace');
  page.press('#text1', 'Backspace');
  sleep(1);
}
```

Closes: https://github.com/grafana/xk6-browser/issues/521

### CODE REVIEW

File: tests/element_handle_test.go [hunk 1]
```diff
@@ -359,3 +359,19 @@ func TestElementHandleWaitForSelector(t *testing.T) {
 
 	element.Dispose()
 }
+
+func TestElementHandler_Press(t *testing.T) {
```
Comment (imiric) [inline tests/element_handle_test.go:363]:
```suggestion
func TestElementHandlePress(t *testing.T) {
```

File: tests/frame_test.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFrame_Press(t *testing.T) {
```
Comment (imiric) [inline tests/frame_test.go:9]:
```suggestion
func TestFramePress(t *testing.T) {
```

File: tests/locator_test.go [hunk 3]
```diff
@@ -370,3 +370,19 @@ func TestLocatorElementState(t *testing.T) {
 		})
 	}
 }
+
+func TestLocator_Press(t *testing.T) {
```
Comment (imiric) [inline tests/locator_test.go:374]:
```suggestion
func TestLocatorPress(t *testing.T) {
```

File: tests/page_test.go [hunk 4]
```diff
@@ -684,6 +684,20 @@ func TestPageWaitForNavigationShouldNotPanic(t *testing.T) {
 	require.NotPanics(t, func() { p.WaitForNavigation(nil) })
 }
 
+func TestPage_Press(t *testing.T) {
```
Comment (imiric) [inline tests/page_test.go:687]:
```suggestion
func TestPagePress(t *testing.T) {
```

**Review by imiric — APPROVED** [review]
LGTM, just some test name nitpicks.

**Review by inancgumus — APPROVED** [review]
LGTM 👍

**Pull request merged**